### ISSUE TITLE
PSR!

### ISSUE THREAD

**Issue by jced-artem** [silvervola/parserSites#1]
Перед тем, как показать эту "библиотеку" людям и запилить пост на хабре - стоило бы прочитать, что такое PSR и воспользоваться их рекомендациями.
Или, хотя бы, нажать в IDE кнопку Reformat code. Или Вы не пользуетесь IDE, как Вы выразились: "по различным причинам (неповоротливость, слишком много кода для получения конечного результата и прочее) и поэтому решил написать свою библиотеку"?

**Comment by dignityinside** [comment]
Да, было бы не плохо оформить всё как пакет, которые легко можно было подгружать через composer. Сделать это совсем не сложно. Посмотрите примеры, например в моих репозиториях.
